modules/webview.py

```diff
@@ -68,9 +92,22 @@ def create(
     extension: str,
     col_bg: str,
     col_accent: str,
-    col_text: str
+    col_text: str,
+    proxy_config: dict | None,
 ):
     config_qt_flags(debug, software)
+
+    if proxy_config and proxy_config["type"] != QNetworkProxy.ProxyType.NoProxy:
+        proxy = QNetworkProxy()
+        proxy.setType(next(p_type for p_type in QNetworkProxy.ProxyType if p_type.name == proxy_config["type"]))
```

Comment:
if this is an enum, you can do `QNetworkProxy.ProxyType[proxy_config["type"]]` to get an enum member by name

Reply:
Seemed to work for me